### DISCUSSION TITLE
running locally

### DISCUSSION THREAD

**Discussion by matuck** [nickcoutsos/keymap-editor#87] [Q&A] [answered]
I think the docs need some updates.  I had to either pass in parameters like app_base_url. or copy .env.tempate to .env.

Second and the bigger issue.
When I run locally and used my cloned repo that I use on the hosted version it does not load my keymap file.  sofle.keymap
Am i dont something wrong?

**Comment by nickcoutsos** [comment, marked as answer]
You're right, the version here is pretty old and I've probably overlooked some things. There are some incompatibilities, between the two versions as well; this branch doesn't include support for parsing devicetree and instead relies on a `keymap.json` file (which the hosted version no longer supports) so you won't be able to conveniently switch between the two right now.

**Reply by matuck** [reply]
I wasn't really wanting to run locally.  I was hoping to help contribute and maybe do a few pull requests.  Do you have the other code somewhere else I would love to help out.

**Reply by nickcoutsos** [reply]
Oh! I appreciate the offer, but I'm not really ready to share the code right now. With the work I'm doing right now things are getting moved around a lot and it's a bit messy. I'm working on breaking it down into smaller packages to help me move the behavior editing features forward but that could be quite a ways away still.
